src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -121,3 +128,14 @@ std::string TriggeredSubscription::toString(const std::string& delimiter)
 
   return ss.str();
 }
+
+
+
+/* ****************************************************************************
+*
+* TriggeredSubscription::stringFilterSet - 
+*/
+void TriggeredSubscription::stringFilterSet(StringFilter* _stringFilterP)
```

Comment:
If this is set with the cache pointer and not actually copied, are we protected from concurrency problems? For example, one update context thread is ussing a TriggeredSubscription while a unsubscribe cache thread is deleting the same csub entry where the pointed StringFilter object lives.